www/medic.js

```diff
@@ -35,30 +35,27 @@ exports.log = function() {
 };
 
 exports.load = function (callback) {
-  var xhr = new XMLHttpRequest();
-  xhr.open("GET", "../medic.json", true);
-  xhr.onload = function() {
-    if (xhr.readyState == 4 && (xhr.status == 0 || xhr.status == 200)) {
-      var cfg = JSON.parse(xhr.responseText);
-      exports.logurl = cfg.couchdb || cfg.logurl;
-      exports.enabled = true;
-      exports.sha = cfg.sha;
-      console.log('Loaded Medic Config: logurl=' + exports.logurl);
-    }
-    callback();
-  }
-  xhr.onerror = function() {
-   callback();
-  }
+  var jsonMedic = null;
 
   try {
+    // attempt to synchronously load medic config
+    var xhr = new XMLHttpRequest();
+    xhr.open("GET", "../medic.json", false);
     xhr.send(null);
-  }
-  catch(ex) {
-    // some platforms throw on a file not found
+    jsonMedic = xhr.responseText;
+  } catch (ex) { }
+
+  // config is available
+  if (jsonMedic) {
+    exports.logurl = JSON.parse(jsonMedic).logurl;
```

Comment:
There is an issue here - earlier code was doing this: exports.logurl = cfg.couchdb || cfg.logurl
while you are only using logurl. In fact, this breaks our medic server